Tasks/DockerV0/container.ts

```diff
@@ -32,6 +32,15 @@ connection.open(tl.getInput("dockerHostEndpoint"), registryAuthenticationToken);
 
 // Run the specified action
 var action = tl.getInput("action", true);
+var telemetry = {
+    registryType: registryType,
+    command: action
+};
+
```

Comment:
No need to make changes for 'Run custom command ' here as there is no arguments field in Version 0 of the task